core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingBlocksEndpointV2.kt

```diff
@@ -308,8 +308,8 @@ fun validatePathfindingResponse(
         }
     }
 
-    val dirTracks = HashSet<Pair<String, EdgeDirection>>()
-    for (track in res.trackSectionRanges) dirTracks.add(Pair(track.trackSection, track.direction))
+    val dirTracks = HashSet<String>()
+    for (track in res.trackSectionRanges) dirTracks.add(track.trackSection)
     if (dirTracks.size != res.trackSectionRanges.size)
```

Comment:
There's a mismatch with the name now

```suggestion
    val trackSet = HashSet<String>()
    for (track in res.trackSectionRanges) trackSet.add(track.trackSection)
    if (trackSet.size != res.trackSectionRanges.size)
```